config/config-authorization.yaml

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-authorization
```

Comment:
Isn't more appropriate to call this config-authentication as we are kind of providing "our" identity?
I guess it does authorization as well, if you don't have permissions in that GSA.
Maybe `config-authnz` and change in other places as well?

Reply:
Makes sense. I think you are right authentication is more appropriate, as that is what we use it for more. But, as you mentioned, it does authorize some WI stuff.

As this is fairly painful to change, let's agree on a name before I make the changes. How about `config-gcp-auth`?

Reply:
`config-auth` or `config-authnz` ? I think I wouldn't add gcp, but that is a personal opinion...
@yolocs as you are our security expert. Any suggestion?

Reply:
Maybe `gcp-auth-config`? My thinking is that "gcp-" would make it clear the config is for auth related stuff with gcp. It could be a little confusing if later we have in cluster (e.g. istio based) auth (not necessarily will happen but just in case).

Reply:
It seems that we should keep the gcp then. 
I'd rather go with `config-gcp-auth` so that we are consistent with the naming of the other configMaps (i.e., starting with config-). But if you guys prefer `gcp-auth-config`, I'm also OK